Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md

```diff
@@ -36,6 +38,22 @@ You can use the Microsoft Authenticator App for both iOS and Android or any othe
 
 ### Phone Authentication
 
-With the phone authenticator, when you log in to the Umbraco Cloud portal, you can choose to either receive an SMS with a code to enter when login in or a call to log you in.
+You have the option to use your phone when you log in to the Umbraco Cloud portal.
+You can choose to receive an SMS with a code or a call to log you in.
```

Comment:
How about using "text message" instead of the acronym here? 🤔

Reply:
Suggestion:
"You can choose to receive a text message with a code or a call to log you in."